client/components/user/user-list.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import store, { getUsersFromServerThunkerator } from '../../store'
+import UserItem from './user-item'
+
+///this should only be available to admin users
+
+export class UserList extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    componentDidMount() {
+        store.dispatch(getUsersFromServerThunkerator());
+    }
+
+    render() {
+        const users = this.props.allUsers;
+        // console.log(users);
+        return (
+            <div>
```

Comment:
class Name allUsers?  something like that

Reply:
created, as discussed we should make generic list and item classes with user and product classes to compound